lms/static/js/spec/views/message_banner_spec.js

```diff
@@ -1,27 +1,46 @@
-define(['backbone', 'jquery', 'underscore', 'js/views/message_banner'
+define(['backbone', 'jquery', 'underscore', 'js/views/message', 'js/common_helpers/template_helpers'
        ],
-    function (Backbone, $, _, MessageBannerView) {
+    function (Backbone, $, _, MessageView, TemplateHelpers) {
         'use strict';
 
-        describe("MessageBannerView", function () {
+        describe("MessageView", function () {
 
             beforeEach(function () {
-                setFixtures('<div class="message-banner"></div>');
+                setFixtures('<div class="message-banner"></div><div class="message"></div>');
```

Comment:
No reason to have two divs with different ids since the element is configurable. The original one will do for both tests.